vuework04/src/views/CheckWalletModal.vue

```diff
@@ -0,0 +1,73 @@
+  <template >
+  <transition name="modal" appear >
+    <div class="overlay" @click.self="$emit('close')" >
+      <div class="content" v-for="user of someUsers" :key="user.id">
+        {{ user.name }}さんの残高: {{ user.wallet }}
+        <footer class="footer">
+          <slot name="footer">
+            <button @click="$emit('close')">閉じる</button>
+          </slot>
+        </footer>
+      </div>
+    </div>
+  </transition>
+</template>
+<script>
+export default {
+  props: ["users"],
+  computed: {
+    someUsers() {
+      return this.users;
+    }
```

Comment:
ここについては親コンポーネントからusers配列とクリックされたusers配列内要素のindex番号を取得するように作り変えます。
```suggestion
  props: ['users', 'index'],
```